src/main/resources/buckets/headBucket.xml

```diff
@@ -20,7 +20,8 @@
     <sequence>
         <class name="org.wso2.carbon.connector.amazons3.auth.RemoveAmazonS3Context"/>
         <property name="uri.var.bucketUrl" expression="$func:bucketUrl"/>
-        <property name="uri.var.uriRemainder" expression="fn:concat('/',$ctx:uri.var.objectName)"/>
+        <!--<property name="uri.var.uriRemainder" expression="fn:concat('/',$ctx:uri.var.objectName)"/>-->
```

Comment:
Remove the commented line

Reply:
Fixed with the commit 61b0d21035122bf4dcaa8e1f053201f1954b12b2